wasmCloud/wasmCloud
# Fix #1724 - TypeScript Hello World example fails to build

## Feature or Problem

The TypeScript Hello World example fails to build on linux while following the doc, the `wash build` command file as the system-linked dependence of `@bytecodealliance/wizer` is fixed to `wizer-darwin-arm64` (for Apple Silicon Macs).

Removing the `package-lock.json` forces the `npm i` to look and recover the right dependence for the system used to build.

## Related Issues

This PR fixes the issue explained in #1724. 

## Release Information

Next release

## Consumer Impact

No impact besides this example

### Manual Verification

To verify it works as intended, you can simple create the project from the example template with `wash new component hello --template-name hello-world-typescript`, delete `package-lock.json`, install with `npm i` and build with `wash build` in any supported system.
